### ISSUE TITLE
Abstract style doesn't apply to nested paragraphs

### ISSUE THREAD

**Issue by ltankey** [AusDTO/gov-au-ui-kit#203]
## What I did

Created a `div.abstract` block.
Nest a paragraph tag beneath it with text.
### What browser and device I was using

Chrome.
## What I expected to see

I expected to see the font-size of the paragraph block nested beneath the abstract tag inherit the same size as text nested directly beneath the abstract block. i.e.

```
%div.abstract
  This text gets the right font-size
  %p
    This text gets the wrong font-size
```
## What I saw

The text nested beneath the paragraph block ended up getting its own font-size set (1rem).
### Additional information (optional)

Nil.

**Comment by TrebBrennan** [comment]
See also issue #101

**Comment by klepas** [comment]
Gotcha.

We can fix this easily, to cater for both `p.abstract` and `div.abstract p`.

Also, I propose we discuss the issue raised in #101, alongside those mentioned by Alex Riolo.

**Comment by TrebBrennan** [comment]
Ticket: DESIGN-262

**Comment by klepas** [comment]
Fixed in 1.7.0.

Deprecation notice: we will be removing `p.abstract` support in ~v2.0, favouring wrapping the abstract instead in a wrapper, eg. `div.abstract`.
